tests/libnet/cluster.go

```diff
@@ -0,0 +1,46 @@
+package libnet
+
+import (
+	"fmt"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	netutils "k8s.io/utils/net"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests/flags"
+)
+
+func IsClusterDualStack(virtClient kubecli.KubevirtClient) (bool, error) {
+	// grab us some neat kubevirt pod; let's say virt-handler is our target.
+	targetPodType := "virt-handler"
+	virtHandlerPod, err := getPodByKubeVirtRole(virtClient, targetPodType)
+	if err != nil {
+		return false, err
+	}
+
+	hasMultipleIPAddrs := len(virtHandlerPod.Status.PodIPs) > 1
```

Comment:
can we combine it with the if in the next line please ?

Reply:
To do that, I'd need to use parenthesis, which to me is a direct indication that what I have now reads better.

Reply:
`if len(virtHandlerPod.Status.PodIPs) != 2`
would it work ?
(imho even if we need parenthesis it would be better)

Reply:
It would, but I think > 1 is what we should be checking. 

If / when eventually we support multiple IPs per iface, that stops working, and future us must adapt it.

Bottom line, this is a style-only comment. You prefer the wall painted blue, I prefer the wall painted white. None of us is right. I just hope none of us block because of this.

Reply:
you saw i didnt block (the lgtm is still there), it doesnt mean its not important imo, just it means that i dont want to block, and leave the decision for you.

since its for the primary interface, i dont think k8s going to change it for more than 1 IP per family,
and even if they do, i prefer this one == 2, in order for us to know they changed the behaviour, since we are checking here a pod which is controller by k8s and CRI.